### PR TITLE
Updated more app helpers

### PR DESCRIPTION
**Pull request by rpranata**
@makdad おなしゃす

### CODE REVIEW

File: Core/LWEUniversalAppHelpers.h [hunk 1]
```diff
@@ -98,6 +98,10 @@ typedef enum
  */
 + (CGFloat)screenHeightDifferenceFrom4InchDisplay;
 
++ (CGFloat)ratioHeightDifferenceFrom4InchDisplay;
+
++ (CGFloat)scaledInBiggerIphoneFor:(CGFloat)number;
```
Comment (makdad) [inline Core/LWEUniversalAppHelpers.h]: comments?

**Comment by makdad** [comment]
looks good but I just want a comment saying what this method does

**Comment by rpranata** [comment]
@makdad Added comment. I thought the method name should be pretty self documenting.

**Comment by makdad** [comment]
Rendy, thanks.  Yeah I know I'm noisy about that sometimes.

**Comment by rpranata** [comment]
:+1: It looks good with comment anyway :)


---
Best Regards,
Rendy Pranata

**Pull request merged**